packages/battery/test/battery_e2e.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:battery/battery.dart';
+import 'package:e2e/e2e.dart';
+
+void main() {
+  E2EWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Builds and runs', (WidgetTester tester) async {
```

Comment:
This is a nit and we're not doing it consistently in first-party unit tests but since this is supposed to be a canonical example of an integration tested plugin, I'm wondering if you think we should wrap the test in a `group('$Battery', () { ... });`

And the description of the test could be `'level'` since that's the method you're invoking.

https://github.com/flutter/plugins/blob/master/packages/image_picker/test/image_picker_test.dart

Reply:
I don't think `group('Battery')` adds much as all tests in this file belong to that group...

Reply:
(renamed the test to 'Can get battery level')